### PR TITLE
Feature/kp/dashboard design

### PR DESCRIPTION
**Pull request by kuldip-trentium**
Screenshot_20240902-110235_CRM
Screenshot_20240902-142336

### CODE REVIEW

File: app/(protected)/(tabs)/tabs.style.ts [hunk 1]
```diff
@@ -59,6 +61,16 @@ export const AddLeadContainer = styled(View)`
...
   flex: 1;
 `;
+export const GreetingText = styled(Text)`
+  font-size: 36px;
+  color: ${({ theme }) => theme.colors.textDark};
+  font-weight: 600;
+`;
+export const NameText = styled(Text)`
+  font-size: 36px;
+  color: #5892ff;
```
Comment (Deeptrentium) [inline app/(protected)/(tabs)/tabs.style.ts:71]: use theme

**Review by Deeptrentium — CHANGES_REQUESTED** [review]

**Review by Deepsutariya149 — APPROVED** [review]

**Review by Deeptrentium — APPROVED** [review]

**Review by sagark1510 — CHANGES_REQUESTED** [review]

**Review by sagark1510 — APPROVED** [review]

File: app/(protected)/(tabs)/dashboard.tsx [hunk 2]
```diff
@@ -121,65 +126,28 @@ const Dashboard = () => {
...
-  };
 
-  const RenderComponent = ({
-    item,
+  const renderLeads = ({
+    lead,
     index,
   }: {
-    item: DashboardLeadList;
+    lead: DashboardLeadList;
```
Comment (Deeptrentium) [inline app/(protected)/(tabs)/dashboard.tsx:134]: DashboardLeadList = DashboardLeadsProps

File: components/molecules/LeadDetail/LeadDetail.styles.ts [hunk 3]
```diff
@@ -8,86 +8,50 @@ export const DetailContainer = styled(View)`
...
-  align-items: center;
-  padding-left: 8px;
-  flex: 1;
-  flex-basis: auto;
-`;
-
-export const NameContainer = styled(View)`
-  display: flex;
-  flex-direction: row;
+export const LeadDetailView = styled(View)`
```
Comment (Deeptrentium) [inline components/molecules/LeadDetail/LeadDetail.styles.ts:12]: use flexed from common style

File: components/molecules/LeadDetail/LeadDetail.tsx [hunk 4]
```diff
@@ -1,123 +1,132 @@
...
+        <ActionModal
+          isModal={showModal}
+          onBackdropPress={hideActionModal}
+          heading={tm('discardMedia')}
+          description={tm('disCardDescription')}
+          label={tm('yesDiscard')}
+          actionType={Actions.delete}
+          actiontext={tm('cancel')}
+          onCancelPress={hideActionModal}
+          onActionPress={() => onDeleteActionPress()}
```
Comment (Deeptrentium) [inline components/molecules/LeadDetail/LeadDetail.tsx:125]: handleDeleteLead()

File: components/molecules/LeadStatus/LeadStatus.styles.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import Text from '@atoms/Text/Text';
+import View from '@atoms/View/View';
+import { styled } from '@utils/styled';
+
+export const LeadStatusContainer = styled(View)<{ bgColor: string }>`
```
Comment (Deeptrentium) [inline components/molecules/LeadStatus/LeadStatus.styles.ts:5]: make it optional

File: components/molecules/LeadsProgressChartItem/LeadsProgressChartItem.tsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  LabelContainer,
+} from '@organisms/LeadsProgressChart/LeadsProgressChart.styles';
+
+const LeadsProgressChartItem: React.FC<LeadsIndicatorItemProps> = ({
+  item,
+  maxValue,
+}) => {
+  const height = useSharedValue(0);
+  const { colors } = useAppTheme();
+  useEffect(() => {
```
Comment (Deeptrentium) [inline components/molecules/LeadsProgressChartItem/LeadsProgressChartItem.tsx:22]: useEffect(() => {
    const containerHeight =
      maxValue && (item.progress / maxValue) * 70;
    height.value = withTiming(containerHeight || 0, { duration: 1000 });
  }, [item.progress, maxValue]);

File: components/molecules/LeadsProgressChartItem/LeadsProgressChartItem.tsx [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+      maxValue === 0 ? 0 : (item.progress / maxValue) * 70;
+    height.value = withTiming(containerHeight, { duration: 1000 });
+  }, [item.progress, maxValue]);
+
+  const animatedStyle = useAnimatedStyle(() => ({
+    height: `${height.value}%`,
+    width: 30,
+    borderRadius: 15,
+    backgroundColor: item?.color,
+    borderWidth: item?.progress !== 0 ? 2 : undefined,
```
Comment (Deeptrentium) [inline components/molecules/LeadsProgressChartItem/LeadsProgressChartItem.tsx:33]: item?.progress && 2 || undefined,

File: components/organisms/LeadsProgressChart/LeadsProgressChart.styles.ts [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+`;
+export const FlatListCon = styled(FlatList).attrs({
+  contentContainerStyle: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    height: 200,
+  },
+})`
+  height: ${({ height }) => `${height}px` || 'auto'};
```
Comment (Deeptrentium) [inline components/organisms/LeadsProgressChart/LeadsProgressChart.styles.ts:37]: its never goes in to ||

File: components/organisms/UserDetailCard/UserDetailCard.styles.ts [hunk 9]
```diff
@@ -41,3 +41,9 @@ export const ViewContainer = styled(View)`
 export const RenderRightView = styled(View)`
   flex-direction: row;
 `;
+export const ActionMenuContainer = styled(View)`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 50;
```
Comment (Deeptrentium) [inline components/organisms/UserDetailCard/UserDetailCard.styles.ts:48]: px missing

File: components/templates/ScreenTemplate/ScreenTemplate.styles.ts [hunk 10]
```diff
@@ -7,7 +7,7 @@ export const Container = styled(SafeAreaView)<{
 }>`
   flex: 1;
   background-color: ${({ backgroundColor, theme }) =>
-    backgroundColor ? backgroundColor : theme.colors.bgColor};
+    backgroundColor ? backgroundColor : theme.colors.screenTemplateColor};
```
Comment (Deeptrentium) [inline components/templates/ScreenTemplate/ScreenTemplate.styles.ts:10]: backgroundColor || theme.colors.screenTemplateColor

**Review by Deeptrentium — CHANGES_REQUESTED** [review]

**Pull request merged**